src/libais/ais8.cpp

```diff
@@ -629,5 +629,34 @@ Ais8_1_31::Ais8_1_31(const char *nmea_payload, const size_t pad)
   status = AIS_OK;
 }
 
+// SSW FI24 Satellite Ship Weather Small - Less than 1-Slot Version
+Ais8_367_24::Ais8_367_24(const char *nmea_payload, const size_t pad)
+    : Ais8(nmea_payload, pad), version(0),
+      utc_hour(), utc_min(), position(), pressure(0.0) {
```

Comment:
Drop the version, utc, pos and pressure initializers here

Reply:
Done